docs/user/03-10-service-description.md

```diff
@@ -74,7 +74,7 @@ These are the provisioning parameters for Azure that you can configure:
 
 | Parameter name | Type | Description | Required | Default value                                 |
 | ---------------|-------|-------------|:----------:|-----------------------------------------------|
-| **machineType** | string | Specifies the provider-specific virtual machine type. | No | `Standard_D8_v3`                              |
+| **machineType** | string | Specifies the provider-specific virtual machine type. | No | `Standard_D4s_v5`                              |
```

Comment:
will this also affect https://help.sap.com/docs/btp/sap-business-technology-platform/provisioning-and-update-parameters-in-kyma-environment?version=Cloud#machine-type ?